pkg/migrations/backfill.go

```diff
@@ -19,19 +19,18 @@ import (
 // 3. Update each row in the batch, setting the value of the primary key column to itself.
 // 4. Repeat steps 2 and 3 until no more rows are returned.
 func backfill(ctx context.Context, conn *sql.DB, table *schema.Table, cbs ...CallbackFn) error {
-	// Get the primary key column for the table
-	pks := table.GetPrimaryKey()
-	if len(pks) != 1 {
-		return errors.New("table must have a single primary key column")
+	// get the backfill column
+	backFillColumn := getBackfillColumn(table)
+	if backFillColumn == nil {
+		return BackfillNotPossible{Table: table.Name}
 	}
-	pk := pks[0]
 
 	// Create a batcher for the table.
 	b := batcher{
-		table:     table,
-		pkColumn:  pk,
-		lastPK:    nil,
-		batchSize: 1000,
+		table:          table,
+		backfillColumn: backFillColumn,
```

Comment:
Maybe `identityColumn` is a better name? `backfillColumn` makes me think it's the column that's going to be backfilled.